charts/generic-govuk-app/templates/transition-config-cron.yaml

```
@@ -0,0 +1,92 @@
+{{- if .Values.transitionConfigEnabled -}}
+{{- $fullName := include "generic-govuk-app.fullname" . }}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ $fullName }}-transition-config-cron
+  labels:
+    {{- include "generic-govuk-app.labels" $ | nindent 4 }}
+    app: {{ $fullName }}-transition-config-cron
+    app.kubernetes.io/name: {{ $fullName }}-transition-config-cron
+    app.kubernetes.io/component: transition-config-cron
+spec:
+  schedule: "00 09-19 * * 1-5"
+  jobTemplate:
+    metadata:
+      name: {{ $fullName }}-transition-config-cron
+      labels:
+        {{- include "generic-govuk-app.labels" $ | nindent 8 }}
+        app.kubernetes.io/component: "{{ .name }}-transition-config-cron"
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          name: {{ $fullName }}-transition-config-cron
+          labels:
+            {{- include "generic-govuk-app.labels" $ | nindent 12 }}
+            app.kubernetes.io/component: transition-config-cron
+        spec:
+          automountServiceAccountToken: false
+          enableServiceLinks: false
+          securityContext:
+            fsGroup: &old-ec2-deploy-uid 2899
+            runAsNonRoot: {{ $.Values.securityContext.runAsNonRoot }}
+            runAsUser: *old-ec2-deploy-uid
+            runAsGroup: *old-ec2-deploy-uid
+          restartPolicy: Never
+          volumes:
+            - name: tmp
+              emptyDir: {}
+            {{- with $.Values.extraVolumes }}
+              {{- . | toYaml | trim | nindent 12 }}
+            {{- end }}
+          initContainers:
+            - name: clone-transition-config-repo
+              image: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/github-cli:latest
+              command: ["/bin/bash"]
+              args: ["-c", "git clone https://github.com/alphagov/transition-config /transition-config/"]
```

It doesn't look like you need bash here. Any reason you can't just run git directly?
```suggestion
              command: ["git"]
              args:
                - clone
                - --depth 1
                - https://github.com/alphagov/transition-config
                - /transition-config/
```
(Can also add `--depth 1` to save time/resources.)

---

this is a good call - let me test this